### ISSUE TITLE
GitHub API - Old Releases Issue

### ISSUE THREAD

**Issue by v00v** [Eloston/ungoogled-chromium#1407]
Why are there no latest package available from "ungoogled-chromium" when accessing it from GitHub API? As you can see by using this API url, it only gives very old version "67.0.3396.87-1" back from "2018-06-19T19:09:55Z". Can you please fix this issue? So those who use github API would be able to pull this data correctly? Thank you!

EDIT-2: It also happens when you try to access this url;
```https://github.com/Eloston/ungoogled-chromium/releases/latest```

Example Command:
```sh
curl -H "Accept: application/vnd.github.v3+json" \
https://api.github.com/repos/eloston/ungoogled-chromium/releases/latest
```

Example Result:
```sh
{
  "url": "https://api.github.com/repos/Eloston/ungoogled-chromium/releases/11555347",
  "assets_url": "https://api.github.com/repos/Eloston/ungoogled-chromium/releases/11555347/assets",
  "upload_url": "https://uploads.github.com/repos/Eloston/ungoogled-chromium/releases/11555347/assets{?name,label}",
  "html_url": "https://github.com/Eloston/ungoogled-chromium/releases/tag/67.0.3396.87-1",
  "id": 11555347,
  "author": {
    "login": "Eloston",
    "id": 1305560,
    "node_id": "MDQ6VXNlcjEzMDU1NjA=",
    "avatar_url": "https://avatars.githubusercontent.com/u/1305560?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/Eloston",
    "html_url": "https://github.com/Eloston",
    "followers_url": "https://api.github.com/users/Eloston/followers",
    "following_url": "https://api.github.com/users/Eloston/following{/other_user}",
    "gists_url": "https://api.github.com/users/Eloston/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/Eloston/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/Eloston/subscriptions",
    "organizations_url": "https://api.github.com/users/Eloston/orgs",
    "repos_url": "https://api.github.com/users/Eloston/repos",
    "events_url": "https://api.github.com/users/Eloston/events{/privacy}",
    "received_events_url": "https://api.github.com/users/Eloston/received_events",
    "type": "User",
    "site_admin": false
  },
  "node_id": "MDc6UmVsZWFzZTExNTU1MzQ3",
  "tag_name": "67.0.3396.87-1",
  "target_commitish": "master",
  "name": "67.0.3396.87-1",
  "draft": false,
  "prerelease": false,
  "created_at": "2018-06-19T19:07:25Z",
  "published_at": "2018-06-19T19:09:55Z",
  "assets": [

  ],
  "tarball_url": "https://api.github.com/repos/Eloston/ungoogled-chromium/tarball/67.0.3396.87-1",
  "zipball_url": "https://api.github.com/repos/Eloston/ungoogled-chromium/zipball/67.0.3396.87-1",
  "body": "# NOTICE: GitHub Releases (i.e. \"Releases\") is no longer used.\r\n\r\n### For *binaries*, [consult this README section](https://github.com/Eloston/ungoogled-chromium/tree/master#download-pre-built-packages).\r\n### For *source code*, [consult this README section](https://github.com/Eloston/ungoogled-chromium/tree/master#getting-the-source-code)\r\n\r\nUpdate to Chromium 67.0.3396.87\r\n* Use bundled FFmpeg for Debian stretch and openSUSE\r\n* Consolidate some Safe Browsing changes"
}
```

Reference: https://docs.github.com/en/rest/reference/repos#releases

**Comment by wchen342** [comment]
It is because there is no release here. You need to pull the tags.

**Comment by Eloston** [comment]
Closed with #1435.
